UnattendResources/Logon.ps1

```diff
@@ -307,6 +351,10 @@ try
 
     Release-IP
 
+    if ($enableAdministrator) {
+        Enable-Administrator

```

Comment:
change the method name to Enable-AdministratorAccount, so that we are consistent with the option name.